src/main/java/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,21 @@
+package report;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String HEADER = "fruit,quantity";
+    private final Map<String, Integer> storage;
+
+    public ReportGeneratorImpl(Map<String, Integer> storage) {
+        this.storage = storage;
+    }
+
+    @Override
+    public String getReport() {
+        return HEADER + System.lineSeparator() + storage.entrySet().stream()
+                .map(entry -> entry.getKey() + "," + entry.getValue())
+                .collect(Collectors.joining(System.lineSeparator()));
+    }
+}
+

```

Comment:
```suggestion
```

Reply:
fix everywhere
if there's one empty line shown on GitHub, it means you have two of them in raw file, GitHub always shows one less